### ISSUE TITLE
Name collisions

### ISSUE THREAD

**Issue by martin-t** [rust-lang/measureme#190]
The name `flamegraph` conflicts with an already existing tool - https://crates.io/crates/flamegraph. I find this confusing - i had the other tool installed (it seems installing one silently overwrites the other) and took me a while to realize what was happening.

I totally wouldn't mind if all the tools had names prefixed with `measureme-`. It would be clear where they came from and since most shells offer decent autocompletion these days it wouldn't be much more extra typing.

**Comment by RalfJung** [comment]
Yeah it would be great if all the measureme tools picked a name that was clearly scoped, rather than just grabbing very general names in the global namespace of binaries.
